python/tk_rv/menu_generation.py

```diff
@@ -86,21 +86,30 @@ def create_menu(self):
         # commands_by_menu = {
         #     "Flow Production Tracking":[menu_item, ...],
         # }
+
         for cmd in menu_commands:
             menu_item = cmd.define_menu_item()
-
             command_added = False
             if menu_overrides:
+
                 for menu_override, commands in menu_overrides.items():
                     app_name = cmd.get_app_name()
-                    if app_name in [
-                        c.get("app_instance")
-                        for c in commands
-                        if cmd.name == c.get("name")
-                    ]:
-                        commands_by_menu[menu_override].append(menu_item)
+                    if not (
+                        menu_override == "SG Review"
+                        and "RV_LOAD_SG_REVIEW" not in os.environ
+                    ):
```

Comment:
Is it possible to "negate" the condition to make it easier to read? Maybe something like:

```suggestion
                    if (
                        menu_override != "SG Review"
                        or "RV_LOAD_SG_REVIEW" in os.environ
                    ):
```

Also, is it possible to flatten the nested conditionals?